core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -327,7 +327,7 @@ class TimeSeriesShard(val ref: DatasetRef,
     * TSP.ingesting is MUCH faster than bit.get(i) but we need the bitmap for faster operations
     * for all partitions of shard (like ingesting cardinality counting, rollover of time buckets etc).
     */
-  private[memstore] final val activelyIngesting = new EWAHCompressedBitmap
+  private[memstore] final val activelyIngesting = debox.Set.empty[Int]
```

Comment:
How is this related to this PR?

Reply:
This was a small omission from previous PR that removed usage of EWAH Bitmap.